### ISSUE TITLE
 Dashboard page: "major" accessibility issues that may not need design 

### ISSUE THREAD

**Issue by ndushay** [sul-dlss/happy-heron#3172]
From https://docs.google.com/spreadsheets/d/1Fs5sY1zgpovWVdHMqpJWRu2Zvyr53jw7, here are "major" accessibility issues for the **Dashboard page**  that we can fix without design changes.   (Note: Mike addressed accessibility level A and AA problems noted by the SiteImprove, WAVE and Axe browser plugins in PR #3159)

Note that solutions may be suggested in https://docs.google.com/spreadsheets/d/1Fs5sY1zgpovWVdHMqpJWRu2Zvyr53jw7.

These can be separate PRs, and/or split into separate tickets.

- [x] ~~The "Cite" button is repeated multiple times: form field labels should be unique.~~ See #3188
- [x] ~~When "Request access to another collection" is selected and the tooltips are visible, the controls that open the tooltips are not associated with the tooltip content.~~ See #3179
- [x] ~~There are link elements that do not have meaningful text.  Within the table, there is link text that is not unique, such as the "Copy persistent link" links.~~ See #3190
- [ ] The error messages appear briefly, but they do not remain visible.  Also in #3171
- [ ] There is no visual indication of keyboard focus when using TAB or SHIFT+TAB to navigate through controls of the website.  This includes the "Deposits in progress -- please complete" and "Your collections" buttons and the Cancel button in the Help modal dialog.  NOTE: may be fixed by PR #3179
- [ ] When the page is resized and hamburger menu is open, focus moves into the links area but focus is not visible.
- [x] ~~Some elements are not well formed.  There are multiple `<h5>`  instances with an id="content-type-prompt" present (id must be unique??)~~ See #3159.
- [x] ~~(Minor level) There are list items that are not direct descendants of list containers (`<ol>` or `<ul>`). For the list of collections, there is an element between the `<ul>` and the `<li>`.~~
- [x] (Minor level) There are headings that have structural markup that does not accurately communicate the heading level.  The modal dialogs each have `<h5>` headings instead of `<h2>`. #See 3332

**Comment by aaron-collier** [comment]
Each of the above points has been resolved, is in progress in a PR, or ticketed separately, closing.
